### ISSUE TITLE
Default Servings per Meal Plan Week

### ISSUE THREAD

**Issue by Wordna1** [grocy/grocy#594]
I apologize if I am posting this in the wrong place.  I am new to Grocy and am very excited about the possibilities.

I am playing around with the demo and can't figure out the following:

Can I define a default number of servings for a given week of a Meal Plan?  We cook for a different number of people each week.

Also, I don't understand why when I click on the shopping cart in the Meal Plan beside "Weekly Costs" I get the following message "Are you sure to put all missing ingredients for recipe "2020-09" on the shopping list?"  Why does it seem to be referring to a date as a recipe?  I'm also not sure what the "09" is referring to since today is 2020-03-07...

**Comment by berrnd** [comment]
> Can I define a default number of servings for a given week of a Meal Plan?

No. But how should that look like? A field above the meal plan table per week?

> Also, I don't understand why when I click on the shopping cart in the Meal Plan beside "Weekly Costs" I get the following message...

The message could be better. Internally the meal plan are just recipes per day and week, that's where it comes from. "2020-09" is week 9 of this year.

**Comment by Wordna1** [comment]
Yeah, a field above the meal plan table per week could definitely work.
